local-cli/generator-windows/index.js

```diff
@@ -23,5 +115,5 @@ module.exports = yeoman.Base.extend({
     this.log(chalk.white('   - or -'));
```

Comment:
I understand, we just don't support those features yet.

---
In reply to: [64813323](https://github.com/ReactWindows/react-native-windows/pull/438#discussion_r64813323) [](ancestors = 64813323)